cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -1748,19 +1751,27 @@ private static void validateOauth(KafkaListenerAuthenticationOAuth oAuth, String
             throw new InvalidResourceException(listener + ": Valid Issuer URI has to be specified");
         }
 
+        if (oAuth.getJwksEndpointUri() != null && (oAuth.getJwksExpirySeconds() != null || oAuth.getJwksRefreshSeconds() != null)) {
```

Comment:
I do not really get it why did you added this condition. I do not think it is needed. You can use JWKS endpoint without specifying expire seconds or refeshs seconds. They have defaults.